specs/assets_overview/02_user_interface.rst

```diff
@@ -0,0 +1,59 @@
+=====================
+User interface
+=====================
+
+=====  ===============  =========  ==========
+epic        title        author     created
+=====  ===============  =========  ==========
+`64`_  Assets overview  tschubotz  2019-01-22
+=====  ===============  =========  ==========
+
+.. _64: https://github.com/gnosis/safe/issues/64
+
+.. _`User interface`:
+
+
+#. `Main`_
+#. `User interface`_
+#. `Technical Details`_
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Assets overview screen
+-------------------------
+
+- Tapping the identicon or the address brings up the receive screen.
+- Tapping "Add token" brings up the manage tokens screen.
```

Comment:
Naming: is it `Add token` or `Add asset`? I think I've asked before, sorry for the repetition.

Reply:
Since you can only tokens at this point, I think "Add tokens" is fine.